app/src/components/menu.ts

```diff
@@ -286,6 +289,43 @@ export function createMenu({
     menuTemplate = [editMenu, viewMenu, windowMenu, helpMenu];
   }
 
+  try {
+    const bookmarkConfigPath = path.join(__dirname, '..', 'bookmarks.json');
+    if (fs.existsSync(bookmarkConfigPath)) {
+      const bookmarksConfig = JSON.parse(
+        fs.readFileSync(bookmarkConfigPath, 'utf-8'),
+      );
```

Comment:
Please also do a little validation that each entry is valid (is either A. a separators, or B. a link with a title & url that can be parsed by `new URL`)

Reply:
I think 1374af0f3a29e0fd63c7de9571d3a904d1ee8e8c addresses this to some extent.